src/install/repository.zig

```diff
@@ -16,21 +16,11 @@ const SloppyGlobalGitConfig = struct {
     }
 
     pub fn loadAndParse() void {
-        const home_dir_path = brk: {
-            if (comptime Environment.isWindows) {
-                if (bun.getenvZ("USERPROFILE")) |env|
-                    break :brk env;
-            } else {
-                if (bun.getenvZ("HOME")) |env|
-                    break :brk env;
-            }
-
-            // won't find anything
-            return;
-        };
+        var home_dir = (bun.os.HomeDir.query(bun.default_allocator)).unwrap() catch return;
```

Comment:
why do we sometimes use env_var.home.get() but here we use HomeDir.query()?

Reply:
Laziness on my part, will fix